src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -2016,7 +2016,11 @@ export abstract class AbstractTaskService extends Disposable implements ITaskSer
 								}
 							}
 							return done(taskSet);
-						}, error);
+						}, error), 5000, () => {
+							// onTimeout
+							console.error('Timed out getting tasks from ', providerType);
+							resolve(result);
```

Comment:
`resolve` could be called multiple times which I think results in an error in the console. Should this be `done` instead, to indicate that the single provider is finished? We only want to resolve when all providers are finished, including _all_ timed out providers